index.html

```diff
@@ -10,7 +10,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap&family=Inter&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/4fd73de0d8.js" crossorigin="anonymous"></script>
-    <title>Portfolio</title>
+    <title>Mike Portfolio</title>
 </head>
 <body>
```

Comment:
- [ ] [OPTIONAL] It seems that you have forgotten to remove the default `margin and padding` of the body, this is why there are some white spaces on each side of your website, you can set the `margin: 0; and padding: 0;` to the body to remove that space. 😉